lib/potluck.rb

```diff
@@ -0,0 +1,67 @@
+require "pry"
+
+class Potluck
+
+  attr_reader :date, :dishes
+  attr_accessor :menu
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+    @menu_hash = {}
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def get_all_from_category(food_type)
+    all = @dishes.find_all do |food|
+      food.category == food_type
+    end
+    return all
+  end
+
+  def menu
+      apps = get_all_from_category(:appetizer)
+      entres = get_all_from_category(:entre)
+      desserts = get_all_from_category(:dessert)
+
+      apps = apps.sort_by do |food|
+         food = food.name[0]
```

Comment:
You don't need this assignment here. You also shouldn't be using the first letter of the name. If two foods have the same first letter, we want to keep comparing based on the rest of the letters in the name. This line should be `food.name`